app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -124,12 +124,24 @@ def analyze
                           msg % [options[:image_full_name], options[:docker_image_id][0..11], actual[0..11]],
                           'error')
     end
+
+    collect_compliance_data(image)
+
     image.scan_metadata(SCAN_CATEGORIES,
                         "taskid" => jobid,
                         "host"   => MiqServer.my_server,
                         "args"   => [YAML.dump(scan_args)])
   end
 
+  def collect_compliance_data(image)
+    return unless image_inspector_client.respond_to?(:fetch_oscap_arf)
+
+    open_scap_result = OpenscapResult.new(:container_image => image)
+    open_scap_result.raw = image_inspector_client.fetch_oscap_arf
+
+    open_scap_result.save
```

Comment:
@roliveri the rest of the information gathered by the smart state analysis are stored using `scan_metadata` and `sync_metadata`. Now, these OpenSCAP results seems not to respect the same behavior (directly saved from here), can you ACK, NACK this part? In case you're against it can you suggest the correct solution? Thanks.